packages/admin-ui/src/pages/system/components/Advanced/ChangeDappnodeWebName.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react";
+import Button from "components/Button";
+import { api } from "api";
+import { InputForm } from "components/InputForm";
+import { useSelector } from "react-redux";
+import { getDappnodeName } from "services/dappnodeStatus/selectors";
+import { withToastNoThrow } from "components/toast/Toast";
+
+export function ChangeDappnodeWebName() {
+  const dappnodeWebName = useSelector(getDappnodeName);
+  const [input, setInput] = useState(dappnodeWebName);
+
+  useEffect(() => {
+    setInput(dappnodeWebName);
+  }, [dappnodeWebName]);
+
+  function onChangeDappNodeWebName() {
+    withToastNoThrow(() => api.dappnodeWebNameSet(input), {
+      message: "Setting dappnode name ...",
+      onSuccess: "Regresh the page to see the changes"
+    });
```

Comment:
For the UI to be reactive it should re-fetch the systemInfo after this request. If you try the UI now, it does not update the name after changing it